cityofaustin/atd-vz-data
# Replace lambda function to update city_id

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/12744

This PR addresses the other part of what the lambda function does (first part was addressed here https://github.com/cityofaustin/atd-vz-data/pull/1247) which is updating the `city_id` to be in Austin when a crashes position is updated to be inside of Austin. I tested the current set up and the lambda function is not doing this as it should be.

To determine whether city_id = 22 (Austin) im using the same jurisdictions that the OG lambda functions use, which is (5, 3, 7, 8, 10). 

![image](https://github.com/cityofaustin/atd-vz-data/assets/67130963/587902c1-3ffa-46dc-abfd-5477d2944bf5)

## Testing
**URL to test:** 
Test [staging](https://visionzero-staging.austinmobility.io/editor/#/crashes)

**Steps to test:**
1. Go to a crash, change the city to be outside of Austin on this Details card.
![image](https://github.com/cityofaustin/atd-vz-data/assets/67130963/eae46119-fbc7-4961-8bda-1741f8e1a3da)
2. Now edit the crash coordinates to be somewhere inside of Austin.
4. Refresh the page, the City should now say Austin in the place you originally edited (aka city_id was updated to 22).

Steps for merge:

- [ ] Recreate these triggers/functions in prod
- [ ] Merge at the same time as https://github.com/cityofaustin/atd-vz-data/pull/1247
- [ ] Delete the `notify_hasura_jurisdiction_from_latlon_INSERT` - AFTER INSERT
`notify_hasura_jurisdiction_from_latlon_UPDATE` - AFTER UPDATE triggers
- [ ] Retire the `jurisdiction_from_latlon` hasura event
- [ ] Shut down the lambda functions

---
#### Ship list
